mkosi/__init__.py

```diff
@@ -1753,6 +1753,10 @@ def reuse_cache(state: MkosiState) -> bool:
     else:
         return False
 
+    for p in (final, build):
+        if p.exists() and p.stat().st_uid != 0:
```

Comment:
Wouldn't that stop reusing the cache whenever it is not owned by root, i.e. also when one only ever does unprivileged builds?

Reply:
No because we're in the user namespace so it will appear as if owned by root.

Reply:
Right, sufficiently far down in the stack. A comment for posteriority would probably make sense.